web/frontend/src/pages/About/About.tsx

```diff
@@ -1,135 +1,109 @@
-import React, { useEffect } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, ScrollRestoration } from 'react-router-dom';
```

Comment:
Still want to figure out why it isn't working to include this somewhere in library code, but we can save that for a future PR.